src/actions/_common-action-add-file.js

```diff
@@ -7,6 +7,7 @@ export default function* addFile(data, cfg, plop) {
 	const makeDestPath = p => path.resolve(plop.getDestBasePath(), p);
 
 	var {template} = cfg;
+	const {overwrite} = cfg;
```

Comment:
what do you think about changing this to a `force` attribute? We could then basically make them the same piece of state, just set globally in the CLI or more granularly at the action config level. I'm thinking we could even build this into the common action interface. Just looking for an opinion here, not changes to the PR. Wondering if you had specific reasons for making this attribute naming different from the CLI option that I'm just not seeing.

Reply:
@JustMaier ^

Reply:
@amwmedia sorry for the delay. Holidays... ¯\_(ツ)_/¯

Renaming to `force` for the sake of consitency makes sense. The only reason I had named it `overwrite` was because I felt it made more sense in the context of an `addFile` action.

Reply:
yeah, I can see why. I think we should treat this as "state" which can be set by the action config... or globally by the user. It would be up to each action to decide what "force" means for its use case.

Reply:
I like that idea